assignment/scripts/aboutMe.js

```diff
@@ -1,44 +1,61 @@
 // REQUIRED FEATURES:
 // 1 - Create a variable called `firstName` and assign it the value of your first name
-
+let firstName= 'Brandon'
```

Comment:
Most JavaScript devs will put a space around any operators like `=`, and end their lines with a semicolon.

```js
let firstName = 'Brandon';
```